doc/getting-started/gsg_quick_setup.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+# Copyright (C) 2018 Intel Corporation.
+# SPDX-License-Identifier: BSD-3-Clause
+# This script is aim to setup or upgrade sos & uos automatically.
+
+# switch for download uos function
+skip_download_uos=0
+# turn on if you need to setup uos
+upgrade_uos=0
+
+function print_help()
+{
+echo "Usage:"
+echo "Launch this script as: ./flash.sh -t 28100"
+echo "-t to specify clear linux version for upgrading"
+echo "-p to specify a proxy server"
+echo "-m to use swupd mirror url"
+echo "-u to upgrade uos, if you just upgrade sos using this script, you'd better to manually reboot sos then continue to use this function."
+echo "-s, skip downloading uos; if enabled, you have to download uos img firstly before upgrading, default is off"
+exit 1
+}
+
+function upgrade_sos()
+{
+# setup mirror, proxy env if specified parameters to p & m opt.
+swupd mirror -u
+if [[ -n $mirror ]]; then
+    echo "Add swupd mirror..."
+    swupd mirror -s $mirror
+fi
+unset https_proxy
+if [[ -n $proxy ]]; then
+    echo "Add proxy..."
+    export https_proxy=$proxy
+fi
+echo "disable auto update..."
+swupd autoupdate --disable
+
+# Compare with current clear linux and skip upgrade sos if get the same version.
+if [[ $(echo -e `swupd info` | cut -d' ' -f3) -eq $target_version ]]; then
+    echo "Specified to the same clear linux version, continue to setup sos env."
+else
+    echo "Upgrade clear linux version to : $target_version ..."
+    swupd verify --fix --picky -m $target_version
+fi
+
+# Do the setups if previous process succeed.
+if [[ $? -eq 0 ]]; then
+    echo "Add service-os kernel-iot-lts2018 bundle"
+    swupd bundle-add service-os kernel-iot-lts2018
+
+    echo "mount /dev/sda1 to /mnt"
+    mount /dev/sda1 /mnt
+
+    echo "Add /mnt/EFI/acrn folder"
+    mkdir -p /mnt/EFI/acrn
+    echo "Copy /usr/share/acrn/samples/nuc/acrn.conf /mnt/loader/entries/"
+    cp -r /usr/share/acrn/samples/nuc/acrn.conf /mnt/loader/entries/
+    echo "Copy acrn.efi to /mnt/EFI/acrn"
+    cp -r /usr/lib/acrn/acrn.efi /mnt/EFI/acrn/
+
+    echo "Check ACRN efi boot event"
+    check_acrn_bootefi=`efibootmgr | grep ACRN`
+    if [[ "$check_arcn_bootefi" -ge "ACRN" ]]; then
+        echo "Clean all ACRN efi boot event"
+        efibootmgr | grep ACRN | cut -d'*' -f1 | cut -d't' -f2 | xargs -i efibootmgr -b {} -B
+    fi
+    echo "Check linux bootloader event"
+    number=$(expr `efibootmgr | grep 'Linux bootloader' | wc -l` - 1)
+    if [[ $number -ge 1 ]]; then
+        echo "Clean all Linux bootloader event"
+        efibootmgr | grep 'Linux bootloader' | cut -d'*' -f1 | cut -d't' -f2 | head -n$number | xargs -i efibootmgr -b {} -B
+    fi
+
+    echo "Add new ACRN efi boot event"
+    efibootmgr -c -l "\EFI\acrn\acrn.efi" -d /dev/sda -p 1 -L "ACRN"
+
+    echo "Create loader.conf"
+    rm /mnt/loader/loader.conf && touch /mnt/loader/loader.conf
```

Comment:
How about we at least keep a backup copy of this?
```suggestion
    mv /mnt/loader/loader.conf /mnt/loader/loader.conf.bck && touch /mnt/loader/loader.conf
```